docs/modules/reference/pages/service-assurance/monitors/DhcpMonitor.adoc

```diff
@@ -31,7 +31,7 @@ CAUTION: Make sure no DHCP client is running on the {page-component-title} serve
 | `org.opennms.netmgt.poller.monitors.DhcpMonitor`
 |===
 
-This monitor implements the <<service-assurance/monitors/introduction.adoc#ga-service-assurance-monitors-common-parameters, Common Configuration Parameters>>.
+This monitor implements the <<reference:service-assurance/introduction.adoc#ref-service-assurance-monitors-common-parameters, Common Configuration Parameters>>.
 
 == DhcpMonitor configuration
```

Comment:
Lines 76 and 79 reference image files that did not get migrated from the operation to reference section.

Reply:
@mmahacek Fixed with https://github.com/OpenNMS/opennms/pull/3624/commits/2a32e4bfc549e03f7f6ac900bcc40b24e1511f08